lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -60,7 +60,8 @@
  % endif
 <article class="course${mode_class}">
   <% course_target = reverse('info', args=[unicode(course_overview.id)]) %>
-  <section class="details">
+  <section class="details" aria-labelledby="details-heading">
+      <h2 class="hd hd-2 sr" id="details-heading">${_('Course details')}</h2>
```

Comment:
Exactly-- if practical, I'd like you to add a bok choy a11y test where there are 2 courses on the dashboard.